### PR TITLE
Default to python 3.10

### PR DESCRIPTION
**Pull request by CoMPaTech**

### CODE REVIEW

File: .github/workflows/verify.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@
 name: Latest commit
 
 env:
-  CACHE_VERSION: 6
-  DEFAULT_PYTHON: "3.9"
+  CACHE_VERSION: 7
+  DEFAULT_PYTHON: "3.10"
```
Comment (frenck) [inline .github/workflows/verify.yml:8]: I won't recommend that, as this allows for sneaking in 3.10 features, that break 3.9 unexceptedly.

File: .github/workflows/verify.yml [hunk 2]
```diff
@@ -206,7 +206,7 @@ jobs:
     needs: prepare-test-cache
     strategy:
       matrix:
-        python-version: [3.9, "3.10"]
```
Comment (frenck) [inline .github/workflows/verify.yml:209]: Is the library removing 3.9 support?

File: .github/workflows/verify.yml (hunk 1 above)
Comment (CoMPaTech) [reply .github/workflows/verify.yml:8]: Good call, just as much I'm not sure why it's breaking at black on 3.9 references now ... I'll look at it again with a fresh pair of eyes sometime later today.

File: .github/workflows/verify.yml (hunk 2 above)
Comment (CoMPaTech) [reply .github/workflows/verify.yml:209]: Not intended as such

Comment (CoMPaTech) [reply .github/workflows/verify.yml:209]: Just wanted to move default to 3.10 and still support 3.9 - as it's mainly focussed at HA we wouldn't drop until Core drops support

**Review by bouwew — APPROVED** [review]
LGTM

File: .github/workflows/verify.yml (hunk 2 above)
Comment (frenck) [reply .github/workflows/verify.yml:209]: Its a matrix, there is no default :)

**Pull request closed without merging**